app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,10 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def check_rbac?
```

Comment:
@PanSpagetka please, change the name of the method to `role_allows_feature?`, as we discussed :wink: